packages/esm-patient-chart-app/src/clinical-views/encounter-list/encounter-list-tabs.extension.tsx

```diff
@@ -1,22 +1,26 @@
 import React, { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Tabs, Tab, TabList, TabPanels, TabPanel } from '@carbon/react';
-import { useConfig, useVisit } from '@openmrs/esm-framework';
+import { useConfig } from '@openmrs/esm-framework';
 import { EncounterList } from './encounter-list.component';
 import { getMenuItemTabsConfiguration } from '../utils/encounter-list-config-builder';
 import styles from './encounter-list-tabs.scss';
 import { filter } from '../utils/helpers';
 import { type Encounter } from '../types';
+import { usePatientChartStore } from '@openmrs/esm-patient-common-lib/src';
```

Comment:
```suggestion
import { usePatientChartStore } from '@openmrs/esm-patient-common-lib';
```

For this and all other occurrences